src/tools/cgxset.c

```diff
@@ -298,8 +303,14 @@ int main(int argc, char *argv[])
 			 * v1 to v2 or vice versa
 			 */
 			ret = 0;
-		else if (ret)
+		else if (ret) {
+			free(converted_src_cgroup);
 			goto err;
```

Comment:
This goto `err` too needs to be replaced with `cgroup_free_err`

Reply:
Good catch.  Looking through the function, I don't see the point of the `err` tag.  I propose we delete the `err` tag and have all the goto's go to `cgroup_free_err`.  Thoughts?

Reply:
It was lazier (easier) to delete the `cgroup_free_tag` and have everything goto `err`.  Let me know what you think.

Reply:
> Looking through the function, I don't see the point of the err tag. I propose we delete the err tag and have all the goto's go to cgroup_free_err. Thoughts?

Agreed, there is no point in having an extra label for the error path, given that there are checks for their validity before calling `cgroup_free()` and at the maximum, extra if checks will be performed in the error path.